x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -131,6 +131,7 @@ export class Simulator {
         coreStart={coreStart}
         databaseDocumentID={databaseDocumentID}
         indices={indices}
+        shouldUpdate={false}
```

Comment:
Would you add simulator tests for this feature?

I would recommend keeping this change the same (always pass `false`) and instead have a simulator method to update the prop.